task/rhacs-deployment-check/3.71/rhacs-deployment-check.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: rhacs-deployment-check
+  labels:
+    app.kubernetes.io/version: "3.71"
+  annotations:
+    tekton.dev/tags: security
+    tekton.dev/categories: Security
+    tekton.dev/displayName: "Policy check a deployment with Red Hat Advanced Cluster Security"
+    tekton.dev/platforms: "linux/amd64"
+    tekton.dev/pipelines.minVersion: "0.18.0"
+spec:
+  description: >-
+    Policy check a deployment with Red Hat Advanced Cluster Security.
+
+    This tasks allows you to check a Kubernetes deployment manifest against policies
+    and apply enforcement to fail pipelines.
+  params:
+    - name: rox_central_endpoint
+      type: string
+      description: Name of secret containing the address:port tuple for RHACS Stackrox Central.
+      default: rox-central-endpoint
+    - name: rox_api_token
+      type: string
+      description: Name of secret containing the RHACS StackRox API token with CI permissions.
+      default: rox-api-token
+    - name: deployment
+      type: string
+      description: |
+        Deployment filename to check.
+        Examples: 'deployment.yaml', '$(workspaces.source.path)/base/deployment.yaml'
+      default: '$(workspaces.source.path)/deployment.yaml'
+    - name: output_format
+      description: Results output format (csv | json | junit | table)
+      type: string
+      default: "table"
+    - name: insecure-skip-tls-verify
+      type: string
+      description: |
+        Do not verify TLS certificates.
+
+        When set to "true", skip verifying the TLS certs of the Central endpoint and registry.
+      default: "false"
+  workspaces:
+    - name: source
+  results:
+    - name: check_output
+      description: Output of `roxctl deployment check`
+  steps:
+    - name: rox-deploy-check
+      #image: registry.redhat.io/advanced-cluster-security/rhacs-roxctl-rhel8:3.71
+      image: quay.io/stackrox-io/roxctl:3.71.0
```

Comment:
can we parameterise the image with `quay.io/stackrox-io/roxctl:3.71.0` as the default value. This will enable users to use the image of their choice if they want to

Reply:
good idea. implemented. it did result in a warning from Catlin

Reply:
I think this is resolved.